bigtable/snippets/reads/read_snippets.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+# Copyright 2020, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigtable_reads_row]
+# [START bigtable_reads_row_partial]
+# [START bigtable_reads_rows]
+# [START bigtable_reads_row_range]
+# [START bigtable_reads_row_ranges]
+# [START bigtable_reads_prefix]
+# [START bigtable_reads_filter]
+from google.cloud import bigtable
+
+# [END bigtable_reads_row]
+# [END bigtable_reads_row_partial]
+# [END bigtable_reads_rows]
+# [END bigtable_reads_row_range]
+# [END bigtable_reads_row_ranges]
+# [END bigtable_reads_prefix]
+# [END bigtable_reads_filter]
+
+# [START bigtable_reads_row]
+from google.cloud.bigtable.row_filters import ColumnQualifierRegexFilter, \
```

Comment:
this import is included in bigtable_reads_row but doesn't seem to be used for this snippet?